src/vesting.cairo

```diff
@@ -65,10 +71,11 @@ mod vesting {
             vested_timestamp: u64
         ) {
             let amt_to_vest = self.milestone.read((vested_timestamp, grantee));
-            assert(amt_to_vest != 0, 'no vesting milestone found, or already vested');
+            assert(amt_to_vest != 0, 'nothing to vest');
             assert(get_block_timestamp() > vested_timestamp, 'not yet eligible');
-            IGovernanceTokenDispatcher { contract_address: govtoken_addr }
-                .mint(claimee, u256 { high: 0, low: amt_to_vest });
+            let state = Governance::unsafe_new_contract_state();
+            IGovernanceTokenDispatcher { contract_address: state.get_governance_token_address() }
+                .mint(grantee, u256 { high: 0, low: amt_to_vest });
```

Comment:
Is it not possible to cast u128 into u256?
```suggestion
                .mint(grantee, amt_to_vest.into());
```
Or maybe even change the storage var to `milestone: LegacyMap::<(u64, ContractAddress), u256>` to keep it consistent?

Reply:
Done, no reason to change the storage var imo